app_settings.py

```diff
@@ -1,23 +1,27 @@
 import os
-from sqlalchemy import create_engine
-import sqlahelper
 import oedialect  # noqa pylint: disable=unused-import
 from configobj import ConfigObj
+from sqlalchemy import create_engine
+import sqlahelper
+
 
 from wam import settings  # pylint: disable=import-error
 
 # add local config file
 fred_config = ConfigObj(os.path.join(settings.BASE_DIR, 'WAM_APP_FRED', 'config', 'fred_app.cfg'))
 
+OEP_ACCESS = fred_config['WAM_APP_FRED']['OEP_ACCESS']
```

Comment:
@christian-rli , @jh-RLI - you need to add OEP_ACCESS=OEP (or OEP_DIALECT) in the file config/fred_app.cfg